packages/cardpay-sdk/sdk/constants.ts

```diff
@@ -145,39 +204,27 @@ const networksConstants = {
     merchantUniLinkDomain: MERCHANT_PAYMENT_UNIVERSAL_LINK_HOSTNAME,
     tallyServiceURL: 'https://reward-api.cardstack.com',
     chainId: 100,
-    scheduledPaymentFeeFixedUSD: 0.25,
-    scheduledPaymentFeePercentage: 0.1, //10%
-  },
+  } as GnosisNetworkConstants,
```

Comment:
Casting like this is not very type safe, I'll work on the side in order to improve the type checking,  feel free to merge this PR as I'll be working on top of it and I'll request your review later.